integration-tests/setup/retrieveStackOutputs.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  DescribeStacksCommand,
+  CloudFormationClient,
+  Output
+} from '@aws-sdk/client-cloudformation'
+
+export const retrieveStackOutputs = async (stack: string, region: string) => {
+  const client = new CloudFormationClient({ region: region })
+  const command = new DescribeStacksCommand({ StackName: stack })
+  const response = await client.send(command)
+
+  if (!response.Stacks) {
+    throw new Error(`Stack with name ${stack} not found.`)
+  }
+
+  const outputs = response?.Stacks[0].Outputs
+
+  if (!outputs) {
+    throw new Error('Stack has no outputs')
+  }
+
+  return outputs
+}
+
+export const getOutputValue = (outputs: Output[], name: string) => {
+  return outputs.find((output) => output.OutputKey === name)
+    ?.OutputValue as string
```

Comment:
Is it worth doing a type check here and throwing an error if we've not got what we're expecting?

Reply:
Added some type checks!